src/pybind/mgr/nfs/export.py

```diff
@@ -784,7 +780,8 @@ def _apply_export(
         )
 
         if not old_export:
-            self._create_export_user(new_export)
+            if old_export.fsal.name == NFS_GANESHA_SUPPORTED_FSALS[1]:
+                self._create_export_user(new_export)
```

Comment:
I'm not sure what this call to `_create_export_user` is doing. Can you explain or add a comment?

Reply:
We're creating new user(with `user_id`) for every export & corresponding cephx key. Since we no more this for CephFS based nfs exports I put condition only for RGW ones.

Reply:
Ah, got it. The `NFS_GANESHA_SUPPORTED_FSALS[1]` obfuscates the meaning the line. You might want to add a commend like `# only for rgw` but I won't demand it. Maybe some day we can come back in and use clearer constants like `FSAL_RGW` instead of `NFS_GANESHA_SUPPORTED_FSALS[1]`

Reply:
Sure, I'll add the comment atleast.